rfc6962-bis.xml

```diff
@@ -1227,6 +1242,9 @@ entry specified by <spanx style="verb">start</spanx>.
 	<t>
 	  Because of skew, it is possible the log server will not have any entries between <spanx style="verb">start</spanx> and <spanx style="verb">end</spanx>. In this case it MUST return an empty <spanx style="verb">entries</spanx> array.
 	</t>
+	<t>
+	  In any case, the log server MUST return the latest STH it knows about when <spanx style="verb">start</spanx> is beyond that STH, and also when the last entry returned is the last entry corresponding to that STH. The log server MAY return the latest STH it knows at any other time.
```

Comment:
Two nits:
(1) replace 'and also' with 'or' (if I understand correctly the latest STH MUST be returned when start > tree_size or last entry returned is tree_size).
(2) 'last entry returned' - so that can be smaller than the last entry requested, right?